MdeModulePkg/Universal/Acpi/BootGraphicsResourceTableDxe/BootGraphicsResourceTableDxe.c

```diff
@@ -588,15 +588,15 @@ BootGraphicsDxeEntryPoint (
   //
   // Register notify function to install BGRT on ReadyToBoot Event.
   //
-  Status = gBS->CreateEventEx (
-                  EVT_NOTIFY_SIGNAL,
-                  TPL_CALLBACK,
-                  BgrtReadyToBootEventNotify,
-                  NULL,
-                  &gEfiEventReadyToBootGuid,
-                  &mBootGraphicsReadyToBootEvent
-                  );
-  ASSERT_EFI_ERROR (Status);
+//  Status = gBS->CreateEventEx (

```

Comment:
No, do not modify the generic modules. If you need to remove it, do it in custom module in UEFIpaylaodPkg for example.

Reply:
Change reverted, we don't need it anymore.